### ISSUE TITLE
xslt-config looks for xml2-config in "/home/fornwall/termux..." directory

### ISSUE THREAD

**Issue by theshoals** [termux/termux-packages#360]
When I run `/data/data/com.termux/files/usr/bin/xslt-config --libs`, I receive this warning:

```
1: /data/data/com.termux/files/usr/bin/xslt-config: /home/fornwall/termux/libxslt/tmp/config-scripts/xml2-config: not found
```

I installed libxslt-dev with `apt install libxslt-dev` (and libxml2-dev with `apt install libxml2-dev`).  I wouldn't expect libxslt-config to search the "/home/fornwall/termux..." directory for libxml2-config on my local device.

**Comment by theshoals** [comment]
I manually edited `/data/data/com.termux/files/usr/bin/xslt-config` to work around this issue:

```
-    the_flags="$the_flags -I$includedir `/home/fornwall/termux/libxslt/tmp/config-scripts/xml2-config --cflags`"
+    the_flags="$the_flags -I$includedir `/data/data/com.termux/files/usr/bin/xml2-config --cflags`"

-    the_flags="$the_flags `/home/fornwall/termux/libxslt/tmp/config-scripts/xml2-config --cflags`"
+    the_flags="$the_flags `/data/data/com.termux/files/usr/bin/xml2-config --flags`"
```

Those edits allowed me to run `/data/data/com.termux/files/usr/bin/xslt-config --libs` and `pip install lxml` without any errors.

**Comment by fornwall** [comment]
Thanks for the descriptive error report - this has now been fixed in version `1.1.29-1` of the `libxslt-dev` package (which now also depends on `libxml2-dev`).
